roles/dns_adblocking/tasks/arch_set_nogroup.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Perhaps I could've moved this part to `arch.yml` but then I'd have to rearrange items in `main.yml`.